contracts/HybridCustody.cdc

```diff
@@ -716,6 +718,18 @@ pub contract HybridCustody {
             self.resources <- {}
         }
 
+        /// Returns a capability to this child account's CapabilityFilter
+        ///
+        pub fun getCapabilityFilter(): Capability<&{CapabilityFilter.Filter}> {
```

Comment:
Would it be preferable to borrow a reference to the filter? Same Q for the Factory Manager method below

Reply:
Not opposed at all. Probably in that case it should be an optional return value in case the capability is no longer valid